packages/logrotate/extra/logrotate.json

```diff
@@ -0,0 +1,34 @@
+{
+  "libraries": [
```

Comment:
Should probably actually live in the `mesos` package. The mesos build is what actually builds the logrotate module. The logrotate package just builds the logrotate binary which we do for other reasons.